cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -158,6 +156,8 @@ func (m *asgCache) register(asg *asg) *asg {
 		existing.MixedInstancesPolicy = asg.MixedInstancesPolicy
 		existing.Tags = asg.Tags
 
+		klog.V(4).Infof("Updated ASG %s. min/max/current is %d/%d/%d", asg.AwsRef.Name, existing.minSize, existing.maxSize, existing.curSize)
```

Comment:
This log suggests that ASG was updated on the cloud provider side.

Reply:
Ah right. How about "Updated ASG cache for %s" instead?